src/mscorlib/mscorlib.shared.sources.props

```diff
@@ -753,6 +755,29 @@
     <ThreadingSources Include="$(BclSourcesRoot)\System\Threading\Tasks\TaskToApm.cs" />
     <ThreadingSources Condition="'$(FeatureCominterop)' == 'true'" Include="$(BclSourcesRoot)\System\Threading\Tasks\IAsyncCausalityTracerStatics.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(FeatureCoreFxFileStream)' != 'true'">
+    <FileStreamSources Include="$(BclSourcesRoot)\System\IO\FileStream.cs" />
+    <SafehandleSources Include="$(BclSourcesRoot)\Microsoft\Win32\SafeHandles\SafeFileHandle.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(FeatureCoreFxFileStream)' == 'true'">
+    <FileStreamSources Include="$(CoreFxSourcesRoot)\System\IO\FileStream.cs" />
+    <FileStreamSources Include="$(CoreFxSourcesRoot)\System\IO\FileStream.NetStandard17.cs" />
+    <FileStreamSources Include="$(CoreFxSourcesRoot)\System\IO\Error.cs" />
+    <FileStreamSources Include="$(CoreFxSourcesRoot)\System\IO\StreamHelpers.ArrayPoolCopy.cs" />
+    <SafehandleSources Condition="'$(TargetsUnix)' == 'true'" Include="$(CoreFxSourcesRoot)\Microsoft\Win32\SafeHandles\SafeFileHandle.Unix.cs" />
+    <SafehandleSources Condition="'$(TargetsUnix)' != 'true'" Include="$(CoreFxSourcesRoot)\Microsoft\Win32\SafeHandles\SafeFileHandle.Windows.cs" />
+    <FileStreamSources Condition="'$(TargetsUnix)' == 'true'" Include="$(CoreFxSourcesRoot)\System\IO\FileStream.Unix.cs" />
+    <FileStreamSources Condition="'$(TargetsUnix)' != 'true'" Include="$(CoreFxSourcesRoot)\System\IO\FileStream.Win32.cs" />
+    <FileStreamSources Condition="'$(TargetsUnix)' != 'true'" Include="$(CoreFxSourcesRoot)\System\IO\FileStreamCompletionSource.Win32.cs" />
+    <FileStreamSources Condition="'$(TargetsUnix)' != 'true'" Include="$(CoreFxSourcesRoot)\System\IO\Win32Marshal.cs" />
```

Comment:
My thought is that everything we share should be "common" (or `\Shared\`). We often end up moving code from one assembly to another- trying to match the assembly layout would get pretty messy. We also sometimes move code to the Common folder in CoreFX.

I'll open an issue on sharing code to discuss long-term options. (Assuming there isn't one already.)